lib/restaurant.rb

```diff
@@ -0,0 +1,90 @@
+require_relative "send_sms"
+require 'time'
+
+class Restaurant
+	attr_reader :pizza_menu
+	attr_reader :orders
+
+	def initialize
+		@pizza_menu = [
+			{bianco: 4.99},
+			{diavolo: 5.99},
+			{funghi: 5.00},
+			{pepperoni: 6.00},
+			{vegetarian: 4.99},
+			{calabria: 5.00},
+			{roma: 6.25}, 
+			{fumo: 5.80},
+			{formaggi: 5.80}, 
+			{regina: 6.80},
+			{fiorentina: 5.80},
+			{sicilian: 6.00}
+		]
+
+		@orders = []
+	end
+
+	def add_to_cart(item)
+		@pizza_menu.each do |hash|
+			if hash.has_key?(item.downcase.to_sym)
+				@orders.push(hash)
+			end
+		end
+		"#{item} successfully submitted!"
+	end
+
+	def confirm_order
```

Comment:
To me, it seems like confirm_order and add_to_cart could have been methods of an Order class so that your Restaurant does not have that many responsibilities. But I understand that would make things a bit more complicated. I have seen it with my eyes haha